### ISSUE TITLE
Usability as C library?

### ISSUE THREAD

**Issue by ThomasdenH** [crlf0710/tectonic#22]
Will this repository be usable as a C library? Do all exported definitions have to be FFI compatible?

**Comment by crlf0710** [comment]
I believe the current answer to me is no. @pkgw might say more about it.

**Comment by ThomasdenH** [comment]
Of course a Rust library can be exposed as a C library later on, so this makes sense.

**Comment by pkgw** [comment]
Yes, currently we aren't accessible from C, but it wouldn't necessarily be a huge effort to change that. See tectonic-typesetting/tectonic#246, which starts moving in this direction (but hasn't been updated in a while).
